src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/AuthorizationEndpointClient.kt

```diff
@@ -251,23 +261,10 @@ internal class AuthorizationEndpointClient(
         else response.body<PushedAuthorizationRequestResponseTO.Failure>()
     }
 
-    private fun toNimbus(
```

Comment:
Why removed?

Reply:
Is moved to [AuthorizationDetailsOps.kt](https://github.com/niscy-eudiw/eudi-lib-jvm-openid4vci-kt/pull/4/files/e5dc09cabd56b0689843b61a1d5d6b799d913e27#diff-971adad0fbb432aad6a708957f833aa4682a2ee2a9b9744994e964bd34dda3d0) along with other utility methods that have to do with how authorization_details are embeded as form parameter